doc/source/User_guide/pyaedt_extensions_doc/project/advanced_fields_calculator.rst

```diff
@@ -0,0 +1,70 @@
+Advanced Fields Calculator
+==========================
+
+The Advanced Fields Calculator provides an additional pool of custom expressions appropriate for each solver.
+These expressions sum up on top of the already existing ones available in the AEDT Fields Calculator.
+These two libraries combined together provide a way to conveniently calculate frequently used quantities.
```

Comment:
```suggestion
These expressions can be used with the already existing ones available in the AEDT Fields Calculator.
These two libraries combined together provide a way to conveniently calculate frequently used quantities.
```
Last sentence, the use of "two libraries" is confusing. Can we change to "two calculators"?